src/Illuminate/Foundation/Exceptions/Handler.php

```diff
@@ -350,7 +352,11 @@ protected function renderExceptionWithWhoops(Exception $e)
      */
     protected function whoopsHandler()
     {
-        return (new WhoopsHandler)->forDebug();
+        try {
+            return app(HandlerInterface::class);
+        } catch (BindingResolutionException $e) {
+            return (new WhoopsHandler)->forDebug();
+        }
```

Comment:
can't we use `Container::bound()` here?

```php
if ($this->container->bound(HandlerInterface::class)) {
    return $this->container->make(HandlerInterface::class);
}

return (new WhoopsHandler)->forDebug();
```

Reply:
Yeah, it will be much better than using global helper method

Reply:
Added PR with this change #29635